### DISCUSSION TITLE
Starting a new project, ormar or SQLModels?

### DISCUSSION THREAD

**Discussion by maitham** [encode/databases#427] [Q&A] [answered]
Hey Folks, 

I'm starting a new project, should I be using this library or SQLModels? What are the pros/cons of both?

**Comment by aminalaee** [comment, marked as answer]
Hey @maitham ,

Unless someone has worked with all the projects you mentioned, the comparison wouldn't be accurate.

But I think SQLModel and ormar are more towards `orm`s but `databases` is more like a generic async database driver.

I think maybe you should compare `SQLModel`, `ormar` and `orm`?

**Reply by maitham** [reply]
Apologies I thought I had posted it on ormar's discussion.. oops 😅
